src/port/port.tcl

```diff
@@ -4577,6 +4577,19 @@ proc parse_options { action ui_options_name global_options_name } {
                         # debug implies verbose
                         set ui_options(ports_verbose) yes
                     }
+                    d2 {
+                        set ui_options(ports_debug) yes
+                        set ui_options(ports_debug2) yes
+                        # debug implies verbose
+                        set ui_options(ports_verbose) yes
+                    }
+                    d3 {
```

Comment:
Can potentially flatten the `dX` checks into one additional case, via a regex on argument string - something like `d[2-3]`?

Along with that, we could also set option `ports_debug_x` to the numeric level of what's enabled. That eliminates having to hard-code multiple option checks for the additional levels.